frontend/src/css/styles.css

```diff
@@ -247,7 +262,7 @@ main .spinner .bounce2 {
 #previewer > button {
   margin: 0;
   position: fixed;
-  top: calc(50% + 1.85em);
+  top: 50%;
```

Comment:
To adjust the left and right arrow's position since we don't have headerbar on the top.